### ISSUE TITLE
[3D Print Design] Suggestion: MagSafe case design

### ISSUE THREAD

**Issue by Stormwind99** [VanceVagell/kv4p-ht#25]
Suggestion: Design a version of the case that attaches magnetically with MagSafe compatible Android phone cases instead of using sticky pads (if the magnets won't cause signal interference).

**Comment by Bakermonitor1932** [comment]
I can tackle this in a couple weeks when the 1.9 design is finished.

**Comment by elitegoodguy** [comment]
Yes!  I was going to tackle a quadlock case once I got my PCB, however a magsafe would be more universal. 

Thank you!

**Comment by kensand** [comment]
Here's a quick and dirty model for it; let me know if you have ideas to improve it. KV4P maintainers feel free to copy it for yourselves, or I can make a PR when I get around to it.

https://codeberg.org/kensand/3dModels/src/branch/main/KV4P-HT/Body

![image](https://github.com/user-attachments/assets/7826e65c-ace1-459a-8922-61c0766481e1)

**Comment by Magnum164** [comment]
I looked at the main case and it looks like a solid model if you used the
one I uploaded as a guide. I can check the other files sometime today if
you want as I am trying to catch up on the designs I am behind in from
being sick last week. But if you followed those practices it should be fine
for JCL to print.

**Comment by VanceVagell** [comment]
Closing this issue as there is now a case option as requested (linked above).
